www/doc/1.5/options.html

```diff
@@ -174,6 +174,9 @@ <h2>Rulesets</h2>
     <tr><td>branches</td><td><code>if</code> statement simplification</td></tr>
     <tr><td>special</td><td>The gamma, Bessel, and error functions</td></tr>
     <tr><td>numerics</td><td>Numerical compounds <code>expm1</code>, <code>log1p</code>, <code>fma</code>, and <code>hypot</code></td></tr>
+
+    <tr><td>simplify</td><td>Will be used in simplify phase</td></tr>
+    <tr><td>fp-safe</td><td>Does not alter floating-point result (e.g. x + 0 ⇝ x) </td></tr>
```

Comment:
I don't think we should document these. I can't think of any reason at all any user would ever want to toggle these rulesets.